common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/PublicApiId.java

```diff
@@ -24,18 +24,83 @@
+    public static final String MULTIPLE_ACCOUNT_PCA_ACQUIRE_TOKEN_SILENT_ASYNC_WITH_SCOPES_ACCOUNT_AUTHORITY_CALLBACK = "28";
+
+    // BrokerClientApplication
+    //==============================================================================================
+    public static final String BROKER_ACQUIRE_TOKEN_SILENT_WITH_PARAMETERS_CALLBACK = "29";
+
+    //endregion
+
+    //region Interactive APIs
+
+    // PublicClientApplication
+    //==============================================================================================
+    public static final String PCA_ACQUIRE_TOKEN_WITH_PARAMETERS = "121";
+    public static final String PCA_ACQUIRE_TOKEN_WITH_ACTIVITY_SCOPES_CALLBACK = "122";
+
+    // SingleAccountPublicClientApplication;
+    //==============================================================================================
+    public static final String SINGLE_ACCOUNT_PCA_SIGN_IN = "123";
+    public static final String SINGLE_ACCOUNT_PCA_ACQUIRE_TOKEN_WITH_PARAMETERS = "124";
+    public static final String SINGLE_ACCOUNT_PCA_ACQUIRE_TOKEN_WITH_ACTIVITY_SCOPES_CALLBACK = "125";
+
+    // MultipleAccountPublicClientApplication
+    //==============================================================================================
+    public static final String MULTIPLE_ACCOUNT_PCA_ACQUIRE_TOKEN_WITH_ACTIVITY_SCOPES_LOGINHINT_CALLBACK = "126";
+
+    // BrokerClientApplication
+    //==============================================================================================
+    public static final String BROKER_ACQUIRE_TOKEN_WITH_PARAMETERS_CALLBACK = "127";
+    public static final String BROKER_ADD_ACCOUNT_WITH_ACTIVITY = "128";
+    public static final String BROKER_CHOOSE_ACCOUNT_WITH_ACTIVITY_ACCOUNTNAME = "129";
+
+    //endregion
+
+    // region GET Accounts
+
+    // SingleAccountPublicClientApplication
+    //==============================================================================================
+    public static final String SINGLE_ACCOUNT_PCA_GET_CURRENT_ACCOUNT = "921";
+    public static final String SINGLE_ACCOUNT_PCA_GET_CURRENT_ACCOUNT_ASYNC = "922";
+
+    // MultipleAccountPublicClientApplication
+    //==============================================================================================
+    public static final String MULTIPLE_ACCOUNT_PCA_GET_ACCOUNTS = "923";
+    public static final String MULTIPLE_ACCOUNT_PCA_GET_ACCOUNTS_WITH_CALLBACK = "924";
+    public static final String MULTIPLE_ACCOUNT_PCA_GET_ACCOUNT_WITH_IDENTIFIER = "925";
+    public static final String MULTIPLE_ACCOUNT_PCA_GET_ACCOUNT_WITH_IDENTIFIER_CALLBACK = "926";
+
+    //endregion
+
+    // region REMOVE account
+
+    // SingleAccountPublicClientApplication
+    //==============================================================================================
+    public static final String SINGLE_ACCOUNT_PCA_SIGN_OUT = "927"; //REMOVE
```

Comment:
> `//REMOVE`

Is this an unresolved TODO?

Reply:
No that is just a comment indicating that this actually does a Remove account, since the term sign out was not explicit about what it does. But since I created regions for each get/remove accounts, I think I can remove that comment.

Reply:
I would either remove the comment or provide more context around it -- in the current state it looks like something you want to remove from the code

Reply:
Addressed